cargo-cyclonedx/src/cli.rs

```diff
@@ -37,6 +38,29 @@ pub struct Args {
     #[clap(long = "quiet", short = 'q')]
     pub quiet: bool,
 
+    // The feature selection flags are not mutually exclusive in Cargo,
+    // so we keep the same behavior here too.
+    /// Activate all available features
+    #[clap(long = "all-features")]
+    pub all_features: bool,
+
+    /// Do not activate the `default` feature
+    #[clap(long = "no-default-features")]
+    pub no_default_features: bool,
+
+    /// Space or comma separated list of features to activate
+    #[clap(long = "features", short = 'F')]
+    pub features: Vec<String>,
+
+    /// The target to generate the SBOM for, e.g. x86_64-unknown-linux-gnu
+    #[clap(long = "target")]
+    pub target: Option<String>,
```

Comment:
These need to be added to the README and the default needs to be documented (host platform) in the readme and in here I'd say